src/components/AuthenticationCheck/index.tsx

```diff
@@ -0,0 +1,99 @@
+import * as S from './style';
+import * as SVG from '../../../public/svg';
+import { useEffect, useState } from 'react';
+import { API } from '../../lib/API';
+
+export default function AuthenticationCheck({
+  email,
+  pw,
+  profileImg = '',
+}: {
+  email: string;
+  pw: string;
+  profileImg?: FileList | string;
+}) {
+  const [checkEmail, setCheckEmail] = useState(false);
+
+  useEffect(() => {
+    const SingUp = async (data?: string) => {
+      try {
+        const { request } = await API.post('/auth/signup', {
+          email: email + '@gsm.hs.kr',
+          password: pw,
+          profileUrl: typeof data == 'string' ? data : '',
```

Comment:
```suggestion
          profileUrl: data ? data : null,
```
형우왈 : 빈 문자열 보단 null 이 좋대

Reply:
확인

Reply:
이거는 add suggestion to batch 누르면 바로 적용됨

Reply:
확인

Reply:
...